test/Graphics/Vertex.test.cpp

```diff
@@ -13,45 +13,48 @@ TEST_CASE("[Graphics] sf::Vertex")
         STATIC_CHECK(std::is_copy_assignable_v<sf::Vertex>);
         STATIC_CHECK(std::is_nothrow_move_constructible_v<sf::Vertex>);
         STATIC_CHECK(std::is_nothrow_move_assignable_v<sf::Vertex>);
+        STATIC_CHECK(std::is_aggregate_v<sf::Vertex>);
     }
 
     SECTION("Construction")
     {
-        SECTION("Default constructor")
+        SECTION("Aggregate initialization -- Nothing")
         {
             constexpr sf::Vertex vertex;
             STATIC_CHECK(vertex.position == sf::Vector2f(0.0f, 0.0f));
             STATIC_CHECK(vertex.color == sf::Color(255, 255, 255));
             STATIC_CHECK(vertex.texCoords == sf::Vector2f(0.0f, 0.0f));
         }
 
-        SECTION("Position constructor")
+        SECTION("Aggregate initialization -- Position")
         {
-            constexpr sf::Vertex vertex({1, 2});
+            constexpr sf::Vertex vertex{{1.0f, 2.0f}};
             STATIC_CHECK(vertex.position == sf::Vector2f(1.0f, 2.0f));
             STATIC_CHECK(vertex.color == sf::Color(255, 255, 255));
             STATIC_CHECK(vertex.texCoords == sf::Vector2f(0.0f, 0.0f));
         }
 
-        SECTION("Position and color constructor")
+        SECTION("Aggregate initialization -- Position and color")
         {
-            constexpr sf::Vertex vertex({1, 2}, {3, 4, 5, 6});
+            constexpr sf::Vertex vertex{{1.0f, 2.0f}, {3, 4, 5, 6}};
             STATIC_CHECK(vertex.position == sf::Vector2f(1.0f, 2.0f));
             STATIC_CHECK(vertex.color == sf::Color(3, 4, 5, 6));
             STATIC_CHECK(vertex.texCoords == sf::Vector2f(0.0f, 0.0f));
         }
 
-        SECTION("Position and coords constructor")
+#if __cplusplus >= 202002L
+        SECTION("Aggregate initialization -- Position and coords")
```

Comment:
I don’t believe any CI job is compiling this. I’d rather we not write conditionally compiled C++20 code. It’s sufficient to assert that this is an aggregate type then we can be certain that designated initializers will work.